packages/ra-data-simple-prisma/src/defaultHandler.ts

```diff
@@ -28,12 +29,13 @@ export const defaultHandler = async (
     delete?: DeleteOptions;
     update?: UpdateOptions;
     getList?: GetListOptions;
+    auditLog?: AuditOptions;
```

Comment:
sometimes is audit, sometimes log, sometimes auditLog.... it should be always the same

Reply:
ok what is it? haha

Reply:
auditLog! and AuditLog